requirements-dev.txt

```diff
@@ -9,6 +9,6 @@ mypy
 coverage
 
 # 3rd party packages for test
-ipywidgets
+ipywidgets==8.0.4
```

Comment:
Can we add a comment here about this is temporary and should be unfreezed as long as ipywidgets is fixed?